v2/mongodb-to-googlecloud/docs/MongoDbToBigQueryCDC/README.md

```diff
@@ -153,3 +165,31 @@ gcloud beta dataflow flex-template run ${JOB_NAME} \
         --template-file-gcs-location=${TEMPLATE_IMAGE_SPEC} \
         --parameters mongoDbUri=${MONGODB_HOSTNAME},database=${MONGODB_DATABASE_NAME},collection=${MONGODB_COLLECTION_NAME},outputTableSpec=${OUTPUT_TABLE_SPEC},inputTopic=${INPUT_TOPIC},userOption=${USER_OPTION}
 ```
+
+### UDF:
+
+Below is the test function that takes document as input and returns the documents as it is.
+
+```
+/**
+ * A transform which adds a field to the incoming data.
+ * @param {Document} doc
+ * @return {Document} returnObj
+ */
+ function transform(doc) {
+    var obj = doc;
+    var returnObj = new Object();
+    return returnObj;
+  }
+```
+
+The template has the following optional parameters:
```

Comment:
Same nit as above. Doesn't need to repeat all the same info as in the earlier section.

Reply:
Removed unnecessary lines.